### PR TITLE
Download assets

### PR DESCRIPTION
**Pull request by paulolieuthier**

### CODE REVIEW

**Review by tonidas — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-inloco/actions-upload-assets
```
Comment (t0rr3sp3dr0) [inline README.md:1]: Upload become download? 🤔

File: action.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  - name: Download assets
+    uses: actions/github-script@v5
+    env:
+      AWS_ACCESS_KEY_ID: ${{ steps.aws-credentials.outputs.aws-access-key-id }}
+      AWS_SECRET_ACCESS_KEY: ${{ steps.aws-credentials.outputs.aws-secret-access-key }}
+      AWS_SESSION_TOKEN: ${{ steps.aws-credentials.outputs.aws-session-token }}
+      ASSETS_NAME: ${{ inputs.name }}
+      ASSETS_BASE_PATH: assets/${{ steps.aws-credentials.outputs.subject }}/run:${{ env.GITHUB_RUN_ID }}/attempt:${{ env.GITHUB_RUN_ATTEMPT }}
+    with:
+      script: |
```
Comment (t0rr3sp3dr0) [inline action.yml]: This is essentially calling bash from node, we should use run instead.

**Review by t0rr3sp3dr0 — CHANGES_REQUESTED** [review]
This action should be converted to a composite action. We are essentially calling bash from node.

File: action.yml (hunk 2 above)
Comment (paulolieuthier) [reply action.yml]: Yeah, contrary to the other actions, that makes perfect sense here.

File: README.md (hunk 1 above)
Comment (paulolieuthier) [reply README.md:1]: Initial commit in master was copied from the other action.

**Comment by paulolieuthier** [comment]
@t0rr3sp3dr0 reimplemented in bash script.

**Pull request merged**